p/themes/Ansum/_components.scss

```diff
@@ -0,0 +1,352 @@
+/*=== COMPONENTS */
+/*===============*/
+/*=== Forms */
+// parti dans _forms.scss
+
+/*=== Horizontal-list */
+.horizontal-list {
+    margin: 0;
+    padding: 0.1rem 0;
+
+    .item{
+	vertical-align: middle;
+
+	&:first-child{
+	    padding-left: 0.5rem;
+	}
+
+    }
+}
+
+/*=== Dropdown */
+.dropdown-menu {
+    background: $grey-lighter;
+    margin: 0;
+    font-size: 1rem;
+    text-align: left;
+    padding: 0.5rem 0 1rem 0;
+    border: none;
+    border-radius: 3px;
+
+    -webkit-box-shadow: 0px 6px 8px 0px rgba(0,0,0,0.35);
+    -moz-box-shadow: 0px 6px 8px 0px rgba(0,0,0,0.35);
+    box-shadow: 0px 6px 8px 0px rgba(0,0,0,0.35);
+
+    &::after {
+	content: "";
+	position: absolute;
+	top: -4px;
+	right: 13px;
+	width: 10px;
+	height: 10px;
+	z-index: -10;
+	transform: rotate(45deg);
+	-moz-transform: rotate(45deg);
+	-webkit-transform: rotate(45deg);
+	-ms-transform: rotate(45deg);
+	background: white;
+	// border-top: 1px solid #95a5a6;
+	// border-left: 1px solid #95a5a6;
+    }
+    
+    .dropdown-header {
+	// padding: 0 5px 5px;
+	margin: 1.75rem 0 0.5rem 2rem;
+	font-weight: bold;
+	text-align: left;
+	color: $grey-dark;
+	text-transform: uppercase;
+	letter-spacing: 1px;
+
+	
+    }
+
+    .item{
+	@include transition(all, 0.075s, ease-in-out);
+	a, span, .as-link{
+	    padding: 0 2rem;
+	    line-height: 2.5em;
+	    font-size: 1rem;
+	    color: $main-font-color;
+	}
+	&:hover{
+	    background: $main-first;
+	    color: $white;
+
+	    a, button{
+		text-decoration: none;
+		color: $white;
+	    }
+	}
+	&[aria-checked="true"]{
+	    a::before{
+		font-weight: bold;
+		margin: 0 0 0 -14px;
+	    }
+	}
+    }
+    .input{
+	select, input{
+	    margin: 0 auto 5px;
+	    padding: 2px 5px;
+	    border-radius: 3px;
+	}
+    }
+    .separator {
+	margin: 0.75rem 0;
+	border-bottom: 1px solid $grey-light;
+	// display: none;
+    }
+
+}
+.tree .tree-folder .tree-folder-items .dropdown-menu,
```

Comment:
@patjennings Is there a particular purpose to such duplicate selectors or is it just a minor oversight?

Reply:
well spotted. No particular reason for this.